src/components/CardComponents/Count/CountTag.tsx

```diff
@@ -32,10 +32,10 @@ const CountTag = ({ media, count }: countProps) => {
 
   return (
     <div>
-      {media == 'Article'
```

Comment:
I'm not sure if we can change these! If you look at playlists I think it's disappeared, and I think it's because we hardcoded these;

(if you want, we could clean this up by creating enums/constants to use for comparisons rather than just regular strings)

Reply:
I'm not totally sure what you mean/what that would look like, but I'm a proponent of no hard coded variables! I remember 'Article' was renamed in the backend, so the countag was not showing up on the "See All" pages

Reply:
fixed!